src/main/java/net/maritimeconnectivity/serviceregistry/config/WebSecurityConfig.java

```diff
@@ -182,6 +172,12 @@ protected void configure(HttpSecurity httpSecurity) throws Exception {
                 ).permitAll()
                 .antMatchers(HttpMethod.OPTIONS, "/**").permitAll() // Allow preflight requests
                 .antMatchers(HttpMethod.GET, "/v2/api-docs").permitAll() // Allow request to Swagger file
```

Comment:
I think the endpoint here should be changed to /v3/api-docs as springdoc-openapi uses OpenAPI 3 instead of version 2.

Reply:
OK, fixed

Reply:
Great, looks good to me! :+1: :)

Reply:
I think you can just do the merge then if you don't have any other last minute changes :)

Reply:
Just make a minor fix to disable eureka by default...

Reply:
but yeah I think we can now merge